ccd-definition/FixedLists/CareSupervision/AuthorityFixedList.json

```diff
@@ -979,6 +979,13 @@
     "ListElement": "Southampton City Council",
     "DisplayOrder": 350
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AuthorityFixedList",
+    "ListElementCode": "SOS",
+    "ListElement": "Southend on Sea City Council",
+    "DisplayOrder": 351
+  },
```

Comment:
This is fine.  I might have gone with DisplayOrder 353.  Then if we have to add a new LA between SCC and SOS, its easier.

Reply:
Good idea. Dan suggested this is a rare exercise and no new entry done in 2 years. If there was a new addition, it will only need updating one entry as the next one is 355.